pkgs/development/ocaml-modules/mec/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, fetchzip, buildDunePackage, ocaml
+, zarith, eqaf, bigarray-compat, hex, ff-sig, ff
+, alcotest, bisect_ppx }:
+
+buildDunePackage rec {
+  pname = "mec";
+  version = "0.1.0";
+  src = fetchzip {
+    url = "https://gitlab.com/nomadic-labs/cryptography/ocaml-ec/-/archive/${version}/ocaml-ec-${version}.tar.bz2";
+    sha256 = "sha256-uIcGj/exSfuuzsv6C/bnJXpYRu3OY3dcKMW/7+qwi2U=";
+  };
+
+  propagatedBuildInputs = [
+    eqaf
+    bigarray-compat
+    hex
+    ff-sig
+    ff
+    alcotest
```

Comment:
This looks wrong but is actually correct

Reply:
Is it really correct or is it an upstream bug (that you may report)?

Reply:
It's actually correct, the usage is here: https://gitlab.com/nomadic-labs/cryptography/ocaml-ec/-/blob/master/src/curve/ec_pbt.ml#L21

Reply:
Suprising design.